assembl/views/__init__.py

```diff
@@ -478,6 +479,19 @@ def get_topic(request):
             return adapt_to_html_content(topic_dict.get(opengraph_locale, topic_dict[locale]))
 
 
+def get_tab_title():
+    """Returns the title to be shown in the tab"""
+    from ..auth.util import get_current_discussion
+    discussion = get_current_discussion()
+    if discussion:
+        if discussion.preferences["tab_title"]:
+            return discussion.preferences["tab_title"]
+        else:
```

Comment:
You can remove this else statement, but keep the outermost else statement. The code execution path will hit it if there is no `tab_title` preference.